x11/xorg-server-devel/Portfile

```diff
@@ -1,49 +1,32 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem 1.0
+PortSystem      1.0
 PortGroup       compiler_blacklist_versions 1.0
 
-name		xorg-server-devel
+name            xorg-server-devel
 conflicts       xorg-server
-set my_name	xorg-server
-version		1.19.99.1
-categories	x11 devel
+set my_name     xorg-server
+version         1.20.99.1
```

Comment:
Suggestion: use a version that's based on the version tags, using `git describe`. For instance, commit #fabbaa8d6260da14fe5b7ca0f3f8748af16253a2 gives a decorated version

```
> git describe
xorg-server-1.19.0-636-gfabbaa8d6
```

which can be transformed to `version 1.19.0.636`. That will probably make the chronological relationship between port:xorg-server and port:xorg-server-devel a lot easier to understand.

Reply:
That is an interesting idea, but I just mirrored with what the maintainer did for the 1.19.x branch. I would prefer to stick with that and leave changing the scheme to him.

Note though the command for the branched used here gives

```
Titan ~/Downloads/xserver > git checkout server-1.20-branch
Branch 'server-1.20-branch' set up to track remote branch 'server-1.20-branch' from 'origin'.
Switched to a new branch 'server-1.20-branch'
Titan ~/Downloads/xserver > git describe
xorg-server-1.20.1
```

and that is not a good version to use...